app/src/main/java/org/oppia/android/app/policies/PoliciesFragmentPresenter.kt

```diff
@@ -78,6 +80,12 @@ class PoliciesFragmentPresenter @Inject constructor(
       supportsLinks = true,
       supportsConceptCards = false
     )
+
+    val websitePattern = Pattern.compile(
+      "(http|ftp|https)://([\\w_-]+(?:(?:\\." +
+        "[\\w_-]+)+))([\\w.,@?^=%&:/~+#-]*[\\w@?^=%&/~+#-])"
+    )
+    LinkifyCompat.addLinks(binding.policyDescriptionTextView, websitePattern, "https://")
```

Comment:
Is this something we can add generally to our HtmlParser class? E.g. here: https://github.com/oppia/oppia-android/blob/a2182477dcdda96d8eaeb8815d13d96fd4093380/utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt#L110